plugins/serializers/wavefront/wavefront.go

```diff
@@ -70,80 +69,62 @@ func (s *WavefrontSerializer) Serialize(m telegraf.Metric) ([]byte, error) {
 
 		name = pathReplacer.Replace(name)
 
-		metric := &wavefront.MetricPoint{
-			Metric:    name,
-			Timestamp: m.Time().Unix(),
-		}
-
-		metricValue, buildError := buildValue(value, metric.Metric)
+		metricValue, buildError := buildValue(value, name)
```

Comment:
We always ignore this error - is there any point returning it from `buildValue()`?

Reply:
Seems to be used to determine if the value should be skipped or not.

Reply:
Typically you log on error from the caller, but we log inside the buildValue function instead to avoid noisy log outputs when the value is of type string (which is expected to not work).

Reply:
Thanks for the insight, I just changed this to return a `bool` instead of an `error` since we used the `error` as a bool (nil / not-nil) and this saves an alloc.